packages/artifact/__tests__/download-specification.test.ts

```diff
@@ -99,6 +101,9 @@ const fileEntry2 = createFileEntry(file2Path)
 const fileEntry3 = createFileEntry(file3Path)
 const fileEntry4 = createFileEntry(file4Path)
 const fileEntry5 = createFileEntry(file5Path)
+const fileEntry6 = createFileEntry(file6Path)
+fileEntry6.fileLength = 0 // empty file path
+fileEntry5.fileLength = undefined // one file does not have a fileLength
```

Comment:
Could we name these file entries to be more descriptive? Something like:
```suggestion
const missingLengthFileEntry = createFileEntry(file5Path)
missingLengthFileEntry.fileLength = undefined // one file does not have a fileLength

const emptyFileEntry = createFileEntry(file6Path)
emptyFileEntry.fileLength = 0 // empty file length
```

Reply:
👍 Had to update the names in a few more places